lib/config.yml

```diff
@@ -19,21 +19,48 @@ Rails:
 Bundler/OrderedGems:
   Enabled: false
 
+Layout/AlignParameters:
+  EnforcedStyle: with_fixed_indentation
+
 Layout/CaseIndentation:
   EnforcedStyle: end
 
 Layout/ElseAlignment:
   Enabled: false
 
+Layout/EmptyLinesAroundBeginBody:
+  Enabled: false
+
 Layout/EmptyLinesAroundBlockBody:
   Enabled: false
 
+Layout/EmptyLinesAroundClassBody:
+  Enabled: false
+
 Layout/EmptyLinesAroundExceptionHandlingKeywords:
   Enabled: false
 
+Layout/FirstArrayElementLineBreak:
```

Comment:
I know you said you hate the end line look of this but it would be consistent and slightly easier to add a new element (a weak argument). In your example you are missing the trailing comma we enforce (I think):
```
[ 
  :a,
  :b,]

#vs
[ 
  :a,
  :b,
]

```

I would be in favor of the end break as well or no line break rules.